Cargo.toml

```diff
@@ -31,7 +31,7 @@ rust-version = "1.76.0"
 [workspace.dependencies]
 log = { version = "0.4", default-features = false }
 qlog = { version = "0.13", default-features = false }
-quinn-udp = { version = "0.5.4", default-features = false, features = ["direct-log"] }
+quinn-udp = { version = "0.5.6", default-features = false, features = ["direct-log"] }
```

Comment:
Don't you need to add `fast-Apple-datapath` or whatever I called the feature?

Reply:
`neqo-udp` does not do GSO or `recvmmsg` today. Thus, even if I enable the `fast-apple-datapath` feature, none of the optimizations introduced in https://github.com/quinn-rs/quinn/pull/1993 would be used.

Adding `recvmmsg` shouldn't be difficult. That said, I would prefer doing so in a separate pull request.